nixos/modules/services/hardware/pcscd.nix

```diff
@@ -56,6 +55,8 @@ in
     environment.systemPackages = [ package ];
     systemd.packages = [ (getBin package) ];
 
+    services.pcscd.plugins = [ pkgs.ccid ];
```

Comment:
what is the rationale for not `lib.mkDefault`? wouldn't list merge?

Reply:
Lists are merged only at the same priority list.

Reply:
Ah I understand better your PR message now :)